bokun.yaml

```diff
@@ -6277,32 +6661,46 @@ definitions:
           $ref: '#/definitions/GeoPoint'
   AgendaItemDto:
     properties:
-      body:
+      id:
+        type: integer
+        format: int64
+        description: 'Specifies ID of the agenda item.'
+      index:
+        type: integer
+        format: int32
+        description: 'Not working.'
```

Comment:
It is working. Index defines order of agenda items (in asc order)

Reply:
It was also my thought, however changing agenda items order did not influence this field value anyhow, it stays 0 all the time.

Reply:
Recommendation: file as bug (probably not reindexing)